go/scheduler/api/api.go

```diff
@@ -67,25 +67,30 @@ type CommitteeNode struct {
 type CommitteeKind uint8
 
 const (
+	// KindCompute is a compute committee.
+	KindInvalid CommitteeKind = 0
+
 	// KindExecutor is an executor committee.
-	KindExecutor CommitteeKind = 0
+	KindExecutor CommitteeKind = 1
```

Comment:
While we are here, can you refactor this to be:
* `KindComputeExecutor`
* `KindComputeTxnScheduler`
* `KindComputeMerge`

And have these be grouped together in value space.